test/spec/ViewerSpec.js

```diff
@@ -798,9 +800,15 @@ describe('Viewer', function() {
         try {
           viewer.importXML(xml);
```

Comment:
Must await this.

Reply:
I don't get why we want to await this. In this test we want to see if viewer.importXML throws an exception if we don't pass a callback and Promises are not polyfilled.